src/scripts/common.js

```diff
@@ -1,4 +1,6 @@
 import { ProjectAutoComplete, TagAutoComplete } from './lib/autocomplete';
+/* eslint-disable-next-line import/no-webpack-loader-syntax */
+import togglButtonSVG from '!!raw-loader!./icons/toggl-button.svg';
```

Comment:
Using the `raw-loader` here so we can still use the shared SVG instead of transforming it into a URL. I'm using the inline option because we may want to use it like the other icons later. Please let me know if this is a problem, I can try to set up something on webpack config to handle it.

Reply:
Makes sense to me 👍

Reply:
Yeah, it's fine. Transforming it into a URL wouldn't solve the problem with Todoist's CSP 💯